packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -299,12 +325,27 @@ export abstract class InversifyHttpAdapter<
               }
             }
 
-            return this.#appendHandlerParam(
+            await this.#appendHandlerParam(
               params,
               index,
               param,
               controllerMethodParameterMetadata.parameterType,
             );
+
+            return this.#applyPipeList(
+              params,
+              [
+                ...this.#globalPipeList,
+                ...controllerMethodParameterMetadata.pipeList,
+              ],
+              {
+                methodName: controllerMethodKey,
+                parameterIndex: index,
+                parameterMethodType:
+                  controllerMethodParameterMetadata.parameterType,
+                targetClass: controller,
```

Comment:
You're passing the controller, but we must pass the controller class. There's a `getBaseType` that can be used for the same purpose

Reply:
Ups, mb. Fixed